cfgov/regulations3k/tests/test_documents.py

```diff
@@ -0,0 +1,42 @@
+from unittest import TestCase
```

Comment:
Would we want Django's TestCase here instead of unittest?

Reply:
Good Question, I hadn't considered the differences but upon some research I don't think we need Django's TestCase:

```
Using unittest.TestCase avoids the cost of running each test in a transaction and flushing the database, but if your tests interact with the database their behavior will vary based on the order that the test runner executes them. This can lead to unit tests that pass when run in isolation but fail when run in a suite.
```

This doesn't rely on a database, we are just defining the class and preparing the document in memory, so I think unittest.TestCase makes more sense here